omnibus/files/private-chef-cookbooks/private-chef/recipes/post_11_upgrade_cleanup.rb

```diff
@@ -65,16 +65,3 @@
 end
 
 private_chef_package_cleaner "varnish"
-
-# Remove old PostgreSQL data directory
-#
-# pg_upgrade will helpfully generate a cleanup script which removes
-# the previous database cluster.  This is nice, because we don't have
-# to worry about keeping track of the location of the old cluster.
```

Comment:
It's merely moved to a generic location -- because it's now for the cleanup of *any* postgres upgrade. The comments [there](https://github.com/chef/chef-server/pull/1310/files#diff-1ada408659099d374677458edaaa8211R18) made me think that this script will contain the old data dir, irregardless of whether it is `/var/opt/opscode/postgresql/data` or versioned. (So it wouldn't care about going from 9.1 (unversioned) to 9.2, or 9.1 (unversioned) to 9.2, or 9.2 to 9.6.)

But I can look the left overs in the upgrade pipelines :+1: